storage/imxrt-flash/imxrt-flashsrv.h

```diff
@@ -34,17 +34,37 @@ typedef struct {
 } __attribute__((packed)) flash_i_devctl_t;
 
 
+typedef struct {
+	uint32_t size;
+	uint32_t psize;
+	uint32_t ssize;
+	uint32_t offs;
+} flashsrv_properties_t;
+
+
 typedef struct {
 	int err;
 
-	struct {
-		uint32_t size;
-		uint32_t psize;
-		uint32_t ssize;
-		uint32_t offs;
-	} properties;
+	flashsrv_properties_t properties;
 
 } __attribute__((packed)) flash_o_devctl_t;
 
 
+/*
+ * Device/partition operations
+ *
+ * Includes parameters and callbacks which allow to perform direct operations (read/write/sector erase) on specific
+ * device/partition. Callbacks are initialized by flashsrv. User should provide *fID* inside flashsrv_customIntInit()
+ * call.
+ */
+typedef struct {
+	int fID;
```

Comment:
Looks like `int fID` is not used, callbacks use `uint8_t fID`. Is it used by user code later like in the /*  description */ above, reserved for future or it's redundant struct member ?

Reply:
It should be `uint8_t fID`. My intention was to use it as argument to callback, so the typical use would be:
```
flashsrv_partitionOps_t *gOps;

void flashsrv_customIntInit(flashsrv_partitionOps_t *ops)
{
	ops->fID = PARTITION_ID;
        gOps = ops;
}

int write(size_t offset, const void *data, size_t size){
        gOps->write(gOps->fID, offset, data, size);`
}
```

Reply:
Ack.